utils/encrypt_decrypt_header.py

```diff
@@ -0,0 +1,26 @@
+from operator import truediv
```

Comment:
Can we have some interoperability tests? E.g. encrypt via lua and decrypt via python? Might need functional tests for that and extra dependency though.